### DISCUSSION TITLE
Draft brief for AU/ESA colab session

### DISCUSSION THREAD

**Discussion by christodd-nhsx** [nhs-oa-community/coding-club#2] [General]
ESA x AU 
Python Coding Club Collaboration

A one hour video call session where colleagues from the two teams show and tell innovative and/or interesting ways in which they have used Python in the workplace. 

The presenter should assume basic working knowledge of Python, but no advanced or specialised skills. Sessions should be detailed enough to give the audience an understanding of the topic, should not be tutorials or detailed step by step guides.

Four demos lasting 15 minutes each will be presented, two from each team. 

Each demo should include:
An overview and business need for the project
Walk through the code pointing out any unique or noteworthy elements
A demo of the functionality to the extent possible
A couple of minutes for questions from the audience
Optional - Link to a github repository

Project 1 - Streamlit app by Craig
Project 2 - 
Project 3 - 
Project 4 -

**Comment by craig-shenton** [comment]
Link to the github project: [https://github.com/nhsengland/AIF_Allocation_Tool](https://github.com/nhsengland/AIF_Allocation_Tool)

I will make a very basic guide showing streamlit in action to run through on the day. 

Resources: [NHS Streamlit theme](https://github.com/nhs-pycom/nhs-streamlit-template).
